clouds/databricks/Makefile

```diff
@@ -88,3 +97,13 @@ ci-release-core:
 
 create-headers:
 	$(MAKE) -C libraries/scala create-headers
+
+create-package: clean build
+	echo "- Creating package"
+	rm -rf $(DIST_DIR)
+	mkdir -p $(DIST_DIR)/$(PACKAGE_NAME)
+	cp $(BUILD_DIR)/modules.sql $(DIST_DIR)/$(PACKAGE_NAME)/
```

Comment:
It must include also the JAR file(s) To be able to install everything from the zip